mysql-ballerina/Module.md

```diff
@@ -0,0 +1,106 @@
+
+This module provides the functionality required to access and manipulate data stored in a MySQL database.
+
+**Prerequisite:** Add the MySQL driver JAR as a native library dependency in your Ballerina project.
+This module uses database properties from the MySQL version 8.0.13 onwards. Therefore, it is recommended to use a
+MySQL driver version greater than 8.0.13. Then, once you build the project by executing the `ballerina build`
+command, you should be able to run the resultant by executing the `ballerina run` command.
+
+E.g., The `Ballerina.toml` content.
+Change the path to the JDBC driver appropriately.
+
+```toml
+[package]
+org = "sample"
+name = "mysql"
+version= "0.1.0"
+
+[[platform.java11.dependency]]
+artifactId = "mysql-connector-java"
+version = "8.0.17"
+path = "/path/to/mysql-connector-java-8.0.17.jar"
+groupId = "mysql"
+``` 
+
+### Client
+To access a database, you must first create a
+[mysql:Client](https://ballerina.io/learn/api-docs/ballerina/#/mysql/clients/Client) object.
+The examples for creating a MySQL client can be found below.
+
+#### Creating a Client
+This example shows different ways of creating the `mysql:Client`.
+
+The client can be created with an empty constructor and hence, the client will be initialized with the default properties.
+The first example with the `dbClient1` demonstrates this.
+
+The `dbClient2` receives the host, username, and password. Since the properties are passed in the same order as it is defined
+in the `jdbc:Client`, you can pass it without named params.
+
+The `dbClient3` uses the named params to pass the attributes since it is skipping some params in the constructor.
+Further [mysql:Options](https://ballerina.io/learn/api-docs/ballerina/#/mysql/records/Options)
+is passed to configure the SSL and connection timeout in the MySQL client.
+
+Similarly, the `dbClient4` uses the named params and it provides an unshared connection pool in the type of
+[sql:ConnectionPool](https://ballerina.io/learn/api-docs/ballerina/#/sql/records/ConnectionPool)
+to be used within the client.
+For more details about connection pooling, see the [SQL Module](https://ballerina.io/learn/api-docs/ballerina/#/sql).
+
+```ballerina
+mysql:Client|sql:Error dbClient1 = new ();
+mysql:Client|sql:Error dbClient2 = new ("localhost", "rootUser", "rooPass", 
+                              "information_schema", 3306);
+                              
+mysql:Options mysqlOptions = {
+  ssl: {
+    mode: mysql:SSL_PREFERRED
+  },
+  connectTimeout: 10
+};
+mysql:Client|sql:Error dbClient3 = new (user = "rootUser", password = "rootPass",
+                              options = mysqlOptions);
+mysql:Client|sql:Error dbClient4 = new (user = "rootUser", password = "rootPass",
+                              connectionPool = {maxOpenConnections: 5});
+```
+
+#### Using SSL
+In order to connect the MySQL database using SSL connection, you must add SSL configurations to `mysql:Options` when crating the `dbClient`.
```

Comment:
```suggestion
To connect the MySQL database using an SSL connection, you must add SSL configurations to `mysql:Options` when creating the `dbClient`.
```

Reply:
corrected in https://github.com/ballerina-platform/module-ballerinax-mysql/pull/157/commits/98bb69547c284bf2186bc2a14cbb622cd2fc34ad